db/migrate/20210301151645_drop_sites.rb

```diff
@@ -0,0 +1,9 @@
+class DropSites < ActiveRecord::Migration[6.0]
+  def up
+    drop_table "sites"
+  end
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+  remove_index :planning_applications, name: :index_planning_applications_on_site_id
```

Comment:
This `remove_index`, if needed (I think it gets dropped by the drop_table anyway), should be inside the `up` method.

Reply:
yes the remove index is needed explicitly because for some reason dropping the table left the index alone when I tried it in the past, causing errors on creating applications without it. I'm not sure why that is,maybe it needed to have been created with a dependent destroy 🤔